File: iCovieApi/iCovieApi/Models/Master/AminitiesModel.cs
using System;
using Newtonsoft.Json;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace iCovieApi.Models
{
    public class AminitiesModel
    {
        [JsonProperty(NullValueHandling = NullValueHandling.Ignore)]
        public int id { get; set; }
        
        [JsonProperty(NullValueHandling = NullValueHandling.Ignore)]
        public string aminities { get; set; }

        [JsonProperty(NullValueHandling = NullValueHandling.Ignore)]
        public string shortname { get; set; }

        [JsonProperty(NullValueHandling = NullValueHandling.Ignore)]
        public string icon { get; set; }


        [JsonProperty(NullValueHandling = NullValueHandling.Ignore)]
        public int cid { get; set; }
    }
}

File: iCovieAdmin-Ng/New folder/src/app/core/auth/_models/Masters/aminities.model.ts


export class AminitiesModel {
    id: number;    
    aminities:string;   
    icon:string;   
    cid:number; 
    isCore: boolean = false;    
    clear(): void {
        this.id= undefined;
        this.aminities= '';        
        this.cid = undefined;      
        this.isCore=false;
	}
}

